src/generatNumber.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+function generateNumber() {
+  return Math.floor(Math.random() * (9999 - 1000 + 1) + 1000);
```

Comment:
Pretty sure it can generate numbers with the same digits in it. Moreover, `(9999 - 1000 + 1)` => `(9000)` 
![same](https://user-images.githubusercontent.com/108527147/203055165-d66b6799-0028-423b-83a2-e1001fba251c.JPG)
Task says: "the program randomly generates a number of 4 different digits"